src/accumulator.jl

```diff
@@ -61,8 +61,6 @@ Base.setindex!(ct::Accumulator, x, v) = setindex!(ct.map, x, v)
 
 Base.haskey(ct::Accumulator, x) = haskey(ct.map, x)
 
-Base.keys(ct::Accumulator) = keys(ct.map)
```

Comment:
technically this adds a level of indirection but it should optimize out with SROA.